### PR TITLE
to https, to adaptive, remove duplicates, clean-up unsorted.m3u

### PR DESCRIPTION
**Pull request by link8k**

### CODE REVIEW

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -422,103 +344,79 @@ https://cdn2.zencast.tv:30443/misiones4/live/chunks.m3u8
...
 http://cdn2.ujjina.com:1935/iptvovacion1/liveovacion1tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",Cultura 24 (Opt-2)
-http://vs8.live.opencaster.com/cultura24/_definst_/mp4:cultura24_480/chunklist_w1999901240.m3u8
+https://vs8.live.opencaster.com/cultura24/_definst_/mp4:cultura24_480/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",Turbo Mix Radio TV
 http://167.114.116.212:1935/tvmb/turbomix/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://i.imgur.com/fzsER.png" group-title="",NTN24
 https://rcnhd.cdn.net.co/live04/ntn24_480p.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://www.suram.com.co/wp-content/uploads/2017/04/LOGO-SURAM.png" group-title="",Suram TV
-https://5b3050bb1b2d8.streamlock.net/suramtv/suramtv/chunklist_w1810740705.m3u8
```
Comment (link8k) [inline channels/unsorted.m3u:431]: This exists in https://github.com/iptv-org/iptv/blob/d1d51f0f2283836bc70ff7578d9344c4f4ef4e53/channels/co.m3u#L27 which is why it was removed.

I converted the links to adaptive and then ran "npm run format"

File: channels/af.m3u [hunk 2]
```diff
@@ -10,9 +10,9 @@ https://d10rltuy0iweup.cloudfront.net/ATNNAT/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV
 https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV US
-http://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
+https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/XpR1VvZ.png" group-title="",Kayhan TV
-http://playout395.livestreamingcdn.com:1935/live/Stream1/playlist.m3u8
+https://playout395.livestreamingcdn.com/live/Stream1/playlist.m3u8
```
Comment (themak1985) [inline channels/af.m3u:15]: New link does not work, original link works.

File: channels/al.m3u [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/YaOhPf6.jpg" group-title="",ABC News
-http://176.31.74.164:8081/abcnewslive/abcnews/chunks.m3u8
+http://176.31.74.164:8081/abcnewslive/abcnews/playlist.m3u8
```
Comment (themak1985) [inline channels/al.m3u:3]: New link does not work, original link works.

File: channels/ar.m3u [hunk 4]
```diff
@@ -42,17 +42,17 @@ https://live-edge01.telecentro.net.ar/live/26hd-720/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="News",Canal 26
 https://live-edge01.telecentro.net.ar/live/smil:c26.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://tvonlinegratis.com/wp-content/uploads/2014/07/Canal4-SanJuan-FDG.jpg" group-title="",Canal 4 San Juan
-http://streamyes.alsolnet.com:1935/canal4sanjuan/live/chunklist.m3u8
+https://streamyes.alsolnet.com/canal4sanjuan/live/playlist.m3u8
```
Comment (themak1985) [inline channels/ar.m3u:45]: New link does not work, original link works.

**Review by themak1985 — COMMENTED** [review]
Hi there @link8k, thanks for your contributions, I can tell its a lot of work. I made a few comments and decided to hold off on a full review until I get some clarity from you on some things.

- What is the rationale for moving to https? In my experience, that could cause issues for some people. 
- I also found that many of the links you transitioned to https, you also removed the proxy, which contributed to breaking the link. 
- Why did you change some of the chunk/chunklist links to playlist links? 
- How did you check all your links? I suggest using VLC and also sites like streamtest.in
- If a channel has a 480P and 1080P stream, it is okay to have both if they are labeled appropriately. It's helpful for those with unreliable internet.  The same applies for backups, but I don't believe you removed any of those. 

Thanks again for your contribution, looking forward to your reply.

File: channels/af.m3u (hunk 2 above)
Comment (link8k) [reply channels/af.m3u:15]: Hi, this one works fine in VLC & via streamtest https://streamtest.in/logs/https-playout395-livestreamingcdn-com-live-stream1-playlist-m3u8-95d3da00-de0c-4e68-a3d7-e514cf53cd1c

File: channels/al.m3u (hunk 3 above)
Comment (link8k) [reply channels/al.m3u:3]: Hi, this one also works fine in VLC & via streamtest https://streamtest.in/logs/http-176-31-74-164-8081-abcnewslive-abcnews-playlist-m3u8

File: channels/ar.m3u (hunk 4 above)
Comment (link8k) [reply channels/ar.m3u:45]: This works for me via VLC

**Comment by link8k** [comment]
Hi @themak1985 , thanks for your comments and your previous contributions!

- What is the rationale for moving to https?

Security and a way of minimising duplicates
- I also found that many of the links you transitioned to https, you also removed the proxy, which contributed to breaking the link.

These work for me and via streamlink.in
- Why did you change some of the chunk/chunklist links to playlist links?

-- To convert them to an adaptive playlist, some streams have 10 bitrates and the player should pick based on the connection.
-- You can use "--adaptive-logic=highest" in VLC to force a specific quality
-- playlist.m3u8 is less likely to break, if the stream changes the qualities it offers / updates the streamer software then someone will have to fix the link. e.g. National Geographic Abu Dhabi changed the max quality from 1080p to 1024p. If "chunklist_b4000000_t64MTA4MHA=.m3u8" was specified then that would have broken.
-- Makes the list shorter
How did you check all your links? I suggest using VLC and also sites like streamtest.in

-- I wrote a python script which uses "streamlink"
- If a channel has a 480P and 1080P stream, it is okay to have both if they are labeled appropriately. It's helpful for those with unreliable internet. The same applies for backups, but I don't believe you removed any of those.

The problem is most aren't labelled and it's a guessing game. I don't want to watch a 1080p stream when I'm on mobile.
I think backups are fine as they're from a different source.

File: channels/ar.m3u (hunk 4 above)
Comment (link8k) [reply channels/ar.m3u:45]: I'm happy to revert the https removal if that's what the maintainers want.

**Review by themak1985 — APPROVED** [review]

File: channels/us.m3u [hunk 5]
```diff
@@ -576,17 +564,13 @@ https://kntvlive-f.akamaihd.net/i/kntvb1_1@15530/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/jrWg5qE.jpg" group-title="Local",NBC San Jose CA (KNTV)
 https://kntvlive-f.akamaihd.net/i/kntvb2_1@15530/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/JSCsJc5.png" group-title="",NBCLX
-https://lxlive-lh.akamaihd.net/i/LX_LIVE@148206/index_1080P_av-p.m3u8
```
Comment (themak1985) [inline channels/us.m3u]: Reverting to 1080P Stream

File: channels/us.m3u [hunk 6]
```diff
@@ -619,8 +603,6 @@ https://ott-live.olympicchannel.com/out/u/OC3_1.m3u8
 https://ott-live.olympicchannel.com/out/u/OC4_1.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://sonyvueimgs.akamaized.net/epg-images/channel/25535/olympic_channel_440x330_NETWORK_LARGE_440_330.png" group-title="",Olympic Channel 6
 https://ott-live.olympicchannel.com/out/u/OC5_1.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/XgF6S22.png" group-title="",Omaha | CBS 3 KMTV
```
Comment (themak1985) [inline channels/us.m3u:622]: Working Link

File: channels/us.m3u [hunk 7]
```diff
@@ -664,13 +646,11 @@ https://rbmn-live.akamaized.net/hls/live/590964/BoRB-AT/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/sOVdYJZ.png" group-title="",Revelation TV
 https://rtv.cdn.mangomolo.com/rtv/smil:switch.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",RT
-http://uiptv.do.am/1ufc/702424953/playlist.m3u8
+https://uiptv.do.am/1ufc/702424953/playlist.m3u8
 #EXTINF:-1 tvg-id="RT America-US" tvg-name="RT America-US" tvg-language="English" tvg-logo="https://www.rt.com/static/block/touch-icon/apple-touch-icon-144x144-precomposed.png" group-title="News",RT America
 https://rt-usa.secure.footprint.net/1105.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/sDGZCpL.png" group-title="",SafeTV
 http://18.191.91.130:1935/live/safetv/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/XOqJ7nO.png" group-title="",Savannah | FOX 28 WTGS
```
Comment (themak1985) [inline channels/us.m3u:672]: Revert, Works

**Review by themak1985 — APPROVED** [review]
Made some changes, good to go now.

**Pull request merged**